lab2/Breakout_2_SLO_Creation_in_Grafana_Cloud.md

```diff
@@ -58,31 +58,31 @@ The main difference between ratio and advanced queries is the complexity of the
 
 ```Step 3:``` Set 'Success Metric' to
 ```
-span_metrics_calls_total{service_name="frontend", status_code!="STATUS_CODE_ERROR", span_kind="SPAN_KIND_CLIENT"}
+traces_spanmetrics_calls_total{service="productcatalogservice", status_code!="STATUS_CODE_ERROR", span_kind="SPAN_KIND_SERVER"}
 ``` 
-This metric accounts for the number of service calls from our frontend, to our microservices, which completed without errors.
+This metric stores the number of requests handled by our product catalog service, which completed without errors.
 
 ```Step 4:``` Set 'Total Metric' to 
 ```
-span_metrics_calls_total{service_name="frontend", span_kind="SPAN_KIND_CLIENT"}
+traces_spanmetrics_calls_total{service="productcatalogservice", span_kind="SPAN_KIND_SERVER"}
 ```
-This metric accounts for the total service calls from our frontend, to our microservices.
+This metric stores the **total** number of requests handled by our product catalog service, errors included.
 
 ```Step 5:``` Set 'Grouping' to
 ```
 span_name
 ```
-This label distinguishes between the different operations being invoked from our application's frontend. This is a big differentiator of Grafana SLO: the capability to provide a "group by" label in our SLI definition, which will give us deep insight into which parts of our application are contributing the most to our error burn rate.
+This label distinguishes between the different operations being handled by the product catalog service. This is a big differentiator of Grafana SLO: the capability to provide a "group by" label in our SLI definition, which will give us deep insight into which parts of our application are contributing the most to our error burn rate.
 
 ```Step 6:``` Click 'Run queries'.  
-You should see a auto-generated SLI query and its visual representation.
+You should see an auto-generated SLI query and its visual representation.
 
 ![Define SLI Tab](./images/bookstore.png)
```

Comment:
I think that image is out of date now - it uses `span_metrics` rather than `traces_spanmetrics`